test/geometry/lane_geometry_test.cc

```diff
@@ -342,26 +367,28 @@ std::vector<RBoundsCase> RBoundsCases() {
           // Right LineString with zero elevation.
           LineString3d{{0., 2., 0.}, {10., 2., 10.}} /* left */,
           LineString3d{{0., -2., 0.}, {10., -2., 0.}} /* right */,
-          // centerline: {0, 0, 0}, {5, 0, 0}, {10, 0, 5}
-          // length: 5.*(std::sqrt(2.) + 1.) = 12.071067811865476
-          12.071067811865476 /* length */,
-          {0., 2.5, 5., 7.5, 12.071067811865476} /* p */,
-          {0., 2.928932188134525, 5.85786437626905, 8.786796564403575, 14.142135623730951} /* expected_left_p */,
-          {0., 2.071067811865475, 4.14213562373095, 6.213203435596426, 10.} /* expected_right_p */,
-          {{-2., 2.},
-           {-2.6864458697336402, 2.045478629078747},
-           {-4.1070628975017831, 2.1762194944608613},
-           {-5.4419740342632519, 2.4671732743644945},
-           {-5.3851648071345037, 5.3851648071345037}} /* expected_r_bounds */
+          LineString3d{{0, 0, 0}, {5., 0., 2.5}, {10., 0., 5.}} /* center */,
+          // length: 5.*std::sqrt(5.)
+          11.180339887498949 /* length */,
+          {0., 11.180339887498949 / 2., 11.180339887498949} /* p */,
+          {0., 14.142135623730951 / 2., 14.142135623730951} /* expected_left_p */,
+          {0., 10. / 2., 10.} /* expected_right_p */,
+          {
+              {-2., 2.},
+              {-3.2015621187164243, 3.2015621187164243},
+              {-5.385164807134504, 5.385164807134504},
+          } /* expected_r_bounds */
       },
       {
           // Arc-like lane:
+          // For further information on this test case, see the following link:
+          // https://github.com/maliput/maliput_sparse/pull/46
           /*
-                  ______________
```

Comment:
Is this change on purpose?

Reply:
Yes! It better represents the geometry of the lane.